src/App.js

```diff
@@ -52,6 +52,14 @@ function App() {
     }
     return properties;
   })
+  useEffect(() => {
+
+    try {
```

Comment:
this is not what I mean. Keep `getPokemones` function and inside implement try catch.
In use effect just call `getPokemones` function